examples/cli/discovery.md

```diff
@@ -223,7 +223,7 @@ You can see that both models are discovered, but only `another_pytorch_model` wa
 # Thanks!
```

Comment:
Line 221 > See the [Input Script documentation](https://github.com/groq/mlagility/blob/main/tools_user_guide.md#input-script) for more details.

Reply:
got it!